Source/WebCore/css/html.css

```diff
@@ -304,43 +304,45 @@ caption {
 ul, menu, dir {
     display: block;
     list-style-type: disc;
-    margin-block-start: 1__qem;
```

Comment:
I'd rather do this change in a separate bug, so it can have enough discussion. This quirk has been around for a long time (since 2002? or even earlier): https://github.com/WebKit/WebKit/commit/ad72cdff4371a3c296ab7a7e162620ebdc940325

Reply:
Reversed for time being and raise new [bug](https://bugs.webkit.org/show_bug.cgi?id=256754).